basics/package.json

```diff
@@ -11,7 +11,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint src --fix",
     "lint-ci": "eslint src --max-warnings=0",
-    "build": "tsc"
+    "build": "tsc",
+    "get:name": "echo $npm_package_name",
```

Comment:
I have never tried it, but you could use [npm-version](https://docs.npmjs.com/cli/v6/commands/npm-version) here, too. Maybe this simplifies some of the work you have to do in the script.

Reply:
`npm version` just manages versions. However, I only want to deploy when we adjust the version in the package.json... So you can use `npm version` to bump to a new patch version e.g., but still you have to deploy. So this is more complementary.